src/lib/random_oracle/random_oracle.ml

```diff
@@ -217,8 +216,15 @@ end
 
 let pack_input = pack_input ~project:Field.project
 
+let prefix_to_field (s : string) =
+  let open Tick0 in
+  assert (8 * String.length s < Field.size_in_bits) ;
```

Comment:
Might it be worth adding a comment here saying `8 bits per character`? `8` looks like it could be a magic number at a glance otherwise.